SuperRonanCraft/bootcamp-project-1
# Image cards

## Purpose

Adding Image Card to main section
Adding modal for button
Adding trailer options for modal
Card hover over efect to show movie content

Fixes #25 

## Changes

Please provide an overview of code changes, especially key files and areas of significant change.

- assets/css/results.css
 - assets/js/search-results.js

## Steps to Reproduce or Test

Demonstrate that the pull request is functional. For example: the exact commands to be run and their output; screenshots if the PR modifies the UI.

- clone the branch
- Hover over image to see to content
- Hit red play button see modal
- Test functionalty of the button on videos

## Optional GIF
![ClassSSRecordVideo](https://github.com/SuperRonanCraft/bootcamp-project-1/assets/122707602/1db22550-77ec-4c42-9f24-27f90e487a46)

Self-explanatory.
